<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BtStart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtDebug">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtResize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize window</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CBoxCateg">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>291</width>
      <height>250</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BtEnableAll">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtDisableAll">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable all</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>332</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
